### DISCUSSION TITLE
Documentation - Optimizing - adding Replaygain

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [AzuraCast/AzuraCast#6672] [Ideas]
Buster can you please add using Replaygain to Optimizing? 
 Something like
_If you enable Replaygain then all your files should be tagged correctly with a tool like [loudgain](https://github.com/Moonbase59/loudgain). Liquidsoap tries to extract the Replaygain tag from the file. If there is no tag found Liquisoap calculates the Replaygain what causes a high CPU usages. So tag your files or do not use Replaygain._

I believe many people are not aware of this.

**Comment by BusterNeece** [comment]
Docs updated and linked from the AutoDJ tab of the station profile.
